protos/core.proto

```diff
@@ -116,6 +116,25 @@ message CompilationError {
   string stack = 3;
 }
 
+
+message EvaluationError {
+  message ErrorLocation {
+    int32 line = 1;
+    int32 column = 2;
+  }
+  string message = 1;
+  ErrorLocation error_location = 2;
```

Comment:
leave a comment on this line stating that if no location can be determined, this field will be unset?